addOns/encoder/src/main/java/org/zaproxy/addon/encoder/EncodeDecodeOptionsPanel.java

```diff
@@ -46,25 +46,41 @@ public class EncodeDecodeOptionsPanel extends AbstractParamPanel {
             Constant.messages.getString("encoder.optionspanel.base64.charset");
     private static final String BREAK_LINES_LABEL =
             Constant.messages.getString("encoder.optionspanel.base64.breaklines");
+    private static final String NAME_HASHERS =
+            Constant.messages.getString("encoder.optionspanel.hashers");
+    private static final String HASHERS_LOWERCASE_LABEL =
+            Constant.messages.getString("encoder.optionspanel.hashers.output.lowercase");
 
     private static final String[] CHARSETS = {"ISO-8859-1", "US-ASCII", "UTF-8"};
 
     private JComboBox<String> comboBoxBase64Charset;
 
     private JCheckBox checkBoxBase64DoBreakLines;
+    private JCheckBox checkBoxHashersLowercase;
 
     private JPanel base64Panel;
+    private JPanel hashersPanel;
 
     public EncodeDecodeOptionsPanel() {
         super();
         setName(NAME);
 
         setLayout(new BorderLayout(0, 0));
 
-        final JPanel panel = new JPanel(new BorderLayout());
+        final JPanel panel = new JPanel(new GridBagLayout());
         panel.setBorder(new EmptyBorder(2, 2, 2, 2));
 
-        panel.add(getBase64Panel(), BorderLayout.NORTH);
+        GridBagConstraints gbc = new GridBagConstraints();
+        gbc.gridx = 0;
+        gbc.gridy = 0;
+        gbc.insets = new java.awt.Insets(2, 2, 2, 2);
+        gbc.anchor = GridBagConstraints.NORTH;
+        gbc.fill = GridBagConstraints.HORIZONTAL;
+        gbc.weighty = 0.0D;
+        gbc.weightx = 1.0D;
+        panel.add(getBase64Panel(), gbc);
+        gbc.gridy = 1;
+        panel.add(getHashersPanel(), gbc);
```

Comment:
I couldn't get this sorted out properly:
![image](https://user-images.githubusercontent.com/7570458/199844764-30c6f44b-84b4-45b4-8b02-4bea22f0cf34.png)

Reply:
If I set the putter gbc to anchor North will that address this?

Reply:
I tried a bunch of different things, I can't get this aligned/sized properly.

Reply:
Finally got it:
![image](https://user-images.githubusercontent.com/7570458/201229534-b66e745a-2b98-4621-82bc-d33aa6695515.png)